docs/topics/feed-exports.rst

```diff
@@ -319,3 +320,16 @@ A dict containing the built-in feed exporters supported by Scrapy.
 .. _URI: http://en.wikipedia.org/wiki/Uniform_Resource_Identifier
 .. _Amazon S3: http://aws.amazon.com/s3/
 .. _boto: http://code.google.com/p/boto/
+
+FEED_BATCH_SIZE
+---------------
+
+Default: ``None``
+
+Allows to control the number of item exported in one file.
+
+The storage URI should define parameters to create a new file that get replaced when the feed is
+being created. These parameters are:
+
+ * ``%(start)s`` - gets replaced by a timestamp when the spider is being started
+ * ``%(time)s`` - gets replaced by a timestamp when the batch feed is being created
```

Comment:
Is there a data loss if several batches are created in the same millisecond? I'm not sure, but it may happen if FEED_BATCH_SIZE is low and spider reads data from cache without going to network.